### DISCUSSION TITLE
Getting - Failed to create project from scheme 'Runner'

### DISCUSSION THREAD

**Discussion by janstehno** [codemagic-ci-cd/codemagic-docs#1264] [Q&A] [answered]
Hello, got stuck on this error when building an iOS Flutter app: `Failed to create project from scheme 'Runner', reason: Error getting launch action, target or config missing.` Link to build here: [...] Never happened in other builds of the same app. Also I cannot find any solutions out there. Android building goes through in Android Studio. I was upgrading Flutter (its version) and updated some dependencies versions, but I do not think that this is what the error causes. I also think I got the workflow config right. I redone the whole repo on GitHub (therefore the new workflow config) so there is a possibility that I did not recreate some files I had there. Thank you in advance.

**Comment by icarusdust** [comment]
Hi @janstehno, it has nothing to do with your Android builds, as the error says it is related to your iOS Scheme. Hard to pinpoint a particular place, I would suggest you to check your Runner scheme if it is in place and correctly configured for the build action.

**Reply by janstehno** [reply]
@icarusdust Hard to pinpoint the problem. Thats what I thought. Sorry to bother but as I am on Windows I do really not know where to find the Runner scheme. Could you help me with that?

I also checked codemagic project samples (specially the ios folder) but cannot seem to find any difference there.

**Comment by janstehno** [comment, marked as answer]
So I deleted all files and folders in the **ios** folder except the gitignore file and Assets folder. Then ran the `flutter create .` command and tried the build again. Seems like everything works as intended. There had to be some minor difference between the files in the ios folder I could not spot. Thank you anyways for the answer.
